codecov.yml

```diff
@@ -3,9 +3,8 @@ comment:
   behavior: default
   require_changes: false
 codecov:
-  require_ci_to_pass: no
+  require_ci_to_pass: false
```

Comment:
What are these changes for?

Reply:
the `no` value wasn't allowing codecov to post comments on the PR directly. That fixes it.